SP/main.yaml

```diff
@@ -0,0 +1,122 @@
+sources:
+  IR_IMERG:
+    args:
+      consolidated: true
+    urlpath: /dados/hackathon/IMERG/{{yyyy}}/{{mm}}/*.nc
+    driver: NetCDF
+    parameters:
+      zoom:
+        allowed:
+          - 9
+        default: 9
+        description: zoom resolution of the dataset
+        type: int
+    metadata:
+      title: NCEP/CPC 4km Global (60N - 60S) IR and GPM IMERG (precipitation) Dataset
+      time_start: 2020-01-01T00:00:00
+      time_end: 2021-04-30T23:00:00
+      creator_name: Paola Bueno, Edson Barbosa
+      creator_email: paola.bueno@iag.usp.br, edson@meteoia.com
+      history: Obtained and processed by Paola Bueno, Edson Barbosa.
+      institution: IAG/USP
+  ERA5:
+    args:
+      consolidated: true
+      urlpath: /dados/hackathon/ERA5/{{level}}/*.nc
+    driver: NetCDF
+    parameters:
+      zoom:
+        allowed:
+          - 7
+        default: 7
+        description: zoom resolution of the dataset
+        type: int
+    metadata:
+      title: ERA5 reanalysis
+      time_start: 2020-01-01T00:00:00
+      time_end: 2021-04-30T23:00:00
+      creator_name: Paola Bueno, Edson Barbosa
+      creator_email: paola.bueno@iag.usp.br, edson@meteoia.com
+      history: Obtained and processed by Paola Bueno, Edson Barbosa.
+      institution: IAG/USP
+    summary: |  
+       Contains modified Copernicus Climate Change Service information 2020. Neither the European Commission nor ECMWF is responsible for any use that may be made of the Copernicus information or data it contains.
+       
+  ERA5Land:
+    args:
+      consolidated: true
+      urlpath: /dados/hackathon/ERA5Land/*.nc
+    driver: zarr
+    parameters:
+      zoom:

```

Comment:
It's (formally speaking) ok to have a parameter which you actually don't use in the `urlpath`, but that seems to be a quite uncommon case. Are you planning on extending the catalog to support multiple `zoom` levels?